README.md

```diff
@@ -15,89 +15,75 @@ If you haven’t set up a "cloudwatch2scalyr" lambda function yet, both of these
 
-<div style="text-align:center"><img src="markdown_images/image_4.png" width="800"></div>
+<div style="text-align:center"><img src="markdown_images/image_2.png" width="900"></div>
 <br>
 
-## Supply environment variables
+Next, click the triangle next to "Environment variables" to expand the "Environment variables" section. You’ll need to specify your Scalyr "Write Logs" API key as an environment variable. In order to do this, you’ll need to create a KMS encryption key if you don’t have one in your account already (beyond the scope of this document). Enable the "Enable helpers for encryption in transit" checkbox, then select your encryption key under "KMS key to encrypt in transit" and "KMS key to encrypt at rest."
 
-You’ll need to specify your Scalyr "Write Logs" API key as an environment variable on this screen (scroll down a bit). In order to do this, you’ll need to create a KMS encryption key if you don’t have one in your account already (beyond the scope of this document). Select your encryption key and enter an environment variable:
+Specify your API key by doing the following:
 
 * Enter "SCALYR_WRITE_LOGS_KEY" in the first column (no quotes)
 * Provide your account’s actual Scalyr "Write Logs" API Key in the second column.
 * Click "Encrypt".
 
 Once you’ve done this, you should see something like:
 
-<div style="text-align:center"><img src="markdown_images/image_5.png" width="900"></div>
-<br>
+<div style="text-align:center"><img src="markdown_images/image_3.png" width="900"></div>
 
-There are a couple of additional environment variables you can pass to the code - `USE_ADD_EVENTS_API`, `PARSER_NAME`, and `SCALYR_BASE_URL`. (These should not be encrypted.)
+There are additional environment variables you can pass to the code - `USE_ADD_EVENTS_API`, `PARSER_NAME`, and `SCALYR_BASE_URL`. (These should not be encrypted.)
 * In general, `USE_ADD_EVENTS_API` should be false (and you don’t actually need to specify it unless you want it to be true).
 * `PARSER_NAME` is optional as well and refers to a specific custom parser defined using the Scalyr UI - if not specified, the default parser is used.
-* `SCALYR_BASE_URL` defaults to `https://www.scalyr.com` and selects the Scalyr region to which you want to upload your logs, such as  `https://upload.eu.scalyr.com`.  *Note:* If this parameter is configured incorrectly, you will not get error messages from Scalyr during log upload; AWS logs of this Lambda function will look as if everything is working, but the logs won't show in Scalyr.
+* `SCALYR_BASE_URL` defaults to `https://www.scalyr.com` and selects the Scalyr region to which you want to upload your logs, such as  `https://upload.eu.scalyr.com`.  *Note:* If this parameter is configured incorrectly, you will not get error messages from Scalyr during log upload; AWS logs of this Lambda function will look as if everything is working, but the logs won't be displayed in Scalyr.
```

Comment:
Here I'd list the allowable values (they're already here, https://www.scalyr.com and https://upload.eu.scalyr.com) and just say it has to be one of those two (for now...)